### PR TITLE
Feat/hover on header menu items

### PR DESCRIPTION
**Pull request by daianaszwimer**

### CODE REVIEW

File: client/app/components/authors/HeaderContainer.scss [hunk 1]
```diff
@@ -60,24 +73,25 @@
         line-height: 0;
 
         @include Desktop() {
-            margin: $spacing-16 $spacing-18 0 $spacing-16;
+            margin: $spacing-20 $spacing-10 0 $spacing-8;
```
Comment (antsgar) [inline client/app/components/authors/HeaderContainer.scss:76]: I think to keep the same spacing as before (16px on the left of the pipe and 18px on the right), we need to make this margin $spacing-20 $spacing-6 0 $spacing-6. On the left we'll have 2px + 8px + 6px = 16px, and on the right it'll be 6px + 12 px = 18px

**Review by antsgar — APPROVED** [review]

File: client/app/assets/styles/mixins.scss [hunk 2]
```diff
@@ -9,3 +9,22 @@
...
+      background-image:
+              linear-gradient(to top, $color 50%, transparent 50%);
+      opacity: 0.08;
+    }
+
+    &:hover {
+      color: $color;
+    }
+  }
+}
```
Comment (antsgar) [inline client/app/assets/styles/mixins.scss:30]: Love how the mixin turned out 🙂 👏

**Review by antsgar — APPROVED** [review]

**Comment by atmoio** [comment]
Eager to get this into develop :)

File: client/app/components/authors/HeaderContainer.scss (hunk 1 above)
Comment (daianaszwimer) [reply client/app/components/authors/HeaderContainer.scss:76]: With that margin, I get the feeling that the author name on hover is too close to the pipe `|`. You will find attached the differences between those two margins. We can discuss this later tho @antsgar , there's no rush
Screen Shot 2021-04-05 at 09 36 57
Screen Shot 2021-04-05 at 09 37 32

Comment (daianaszwimer) [reply client/app/components/authors/HeaderContainer.scss:76]: Fixed here https://github.com/standardnotes/listed/pull/150

**Pull request merged**